tox.ini

```diff
@@ -11,10 +11,10 @@ commands =
 [testenv:coverage]
 usedevelop = true
 basepython =
-    python2.7
+    python3.6
 commands =
     coverage run -m zope.testrunner --test-path=src --all
-    coverage report
+    coverage report --fail-under=100
```

Comment:
Can we make travis fail as well if coverage drops under 100%?

(I no longer trust coveralls.io to police coverage levels.  I don't remember why, precisely.)

Reply:
Ha!  https://github.com/zopefoundation/zope.configuration/pull/32 proves me wrong.

Reply:
coveralls failed #32 because it wasn't at 100% coverage :) 

The one issue that I've seen with coveralls not failing something that I would have expected to fail has come down to precision and rounding up to 100%. I think without specifying precision in the .coveragerc we potentially have that problem locally too.